uportal-war/src/main/java/org/apereo/portal/portlets/lookup/PersonLookupHelperImpl.java

```diff
@@ -427,9 +427,9 @@ protected IAuthorizationPrincipal getPrincipalForUser(final IPerson person) {
      * @param attributeNames
      * @return
      */
-    protected Set<String> getAvailableAttributes(
+    protected Set<String> getPermittedAttributes(
```

Comment:
I'm not objecting to the name change. But note that the name change breaks the API in a non-private method. To the extent that anything depended upon this API, it broke with this name change. To the extent that anything depends on this API going forward, it's depending on an API that exists only in uPortal v5 and later, presumably. To the extent that this isn't actually an API that should be exposed beyond private, let's mark it `private` and I won't want it to have an `@since` tag. :smile:

`@since` tags are meta-valuable in that they encourage thinking about what APIs are being exposed and the cost-benefits of those changes. This name change is fine -- the benefit of clearer naming probably does outweigh the cost. At least I think so, and you do too, which is why you changed it. :+1:. But let's acknowledge that it creates an API that exists only where it exists.